packages/react/menu/src/Menu.tsx

```diff
@@ -707,6 +709,7 @@ const MenuSubTrigger = React.forwardRef((props, forwardedRef) => {
           } else {
             // There's 100ms where the user may leave an item before the submenu was opened.
             contentContext.onPointerGraceAreaChange(null);
+            context.parentContent?.focus();
```

Comment:
Would `contentContext.onItemLeave()` work here? I think it does the same thing 💭

My thinking is that we wouldn't need the `context.parentContent` then because `contentContext` should be the same thing from this part.

Reply:
Good point, original thinking was it referenced the submenu content but as that’s raised should have worked here.